core/Policy/PolicyManager.php

```diff
@@ -48,7 +95,7 @@ public static function getPolicyByName(string $policyName): ?string
     /**
      * @return array<class-string<PolicyComparisonInterface<mixed>&SettingValueInterface<mixed>>>
      */
-    public static function getAllSettings(?int $idSite = null): array
+    public static function getAllSettings(?int $idSite = null, ?string $settingType = null): array
```

Comment:
Any reason why this has `$idSite` as param. It currently seems unused.

Reply:
Probably doesn't need it as it's only called internally. Will remove.